configure.ac

```diff
@@ -76,11 +76,11 @@ AC_CHECK_HEADERS(sys/time.h)
 AC_HEADER_SYS_WAIT
 
 AC_CHECK_DECLS(S_IRGRP)
-if test x$ac_cv_have_decl_S_IRGRP = xyes ; then
-	AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP],1,[Set to 1 if S_IRGRP is defined.])
-else
-	AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP],0)
-	fi
+AS_IF([test "x$ac_cv_have_decl_S_IRGRP" = "xyes"], [
+	AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP], [1], [Set to 1 if S_IRGRP is defined.])
+], [
+	AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP], [0], [Set to 0 if S_IRGRP is not defined.])
+])
```

Comment:
I think that idiom is far less clear that the original and find the indentation you have chosen to be less than ideal.

Since this is a common idiom in libsndfile's `configure.ac` I would prefer if we defined a new M4 macro that expands to what you have with improved indenting. Eg:
```
AS_IF([test "x$ac_cv_have_decl_S_IRGRP" = "xyes"], [
    AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP], [1], [Set to 1 if S_IRGRP is defined.])
    ], [
    AC_DEFINE_UNQUOTED([HAVE_DECL_S_IRGRP], [0], [Set to 0 if S_IRGRP is not defined.])
    ])
```